fw-child/languages/cdc/fr_CA.po

```diff
@@ -972,6 +971,14 @@ msgstr ""
 msgid "Article"
 msgstr ""
 
+#: template/learn/query.php:28
+msgid "Title (A–Z)"
+msgstr "Titre (A à Z)"
+
+#: template/learn/query.php:29
+msgid "Title (Z–A)"
+msgstr "Titre (Z à A)"
+
```

Comment:
This file (and the .pot and .mo files) would need to be regenerated since those texts are not used anymore.